index.html

```diff
@@ -4,10 +4,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <title>Hello World!</title>
 </head>
 <body>
-    <header>
```

Comment:
This header should be the one containing the class name

Reply:
It is. In the next commit.